src/whirlwind/_cost.py

```diff
@@ -29,14 +30,26 @@ def calc_smooth_phase_gradients(igram):
     return phase_dy_smooth, phase_dx_smooth
 
 
+def _load_rgi(path: Path | str) -> RegularGridInterpolator:
+    """Reconstruct RegularGridInterpolator from saved data."""
+    data = np.load(path, allow_pickle=False)
+    return RegularGridInterpolator(
+        points=tuple(data["grid"]),
+        values=data["values"],
+        method=str(data["method"]),
+        bounds_error=bool(data["bounds_error"]),
+        fill_value=float(data["fill_value"])
+        if data["fill_value"] is not None
+        else None,
```

Comment:
Looks like your formatter barfed here. Can we fit this on a single line for readability?

Maybe something like

```python
    fill_value = float(data["fill_value"]) if data["fill_value"] is not None else None
    return RegularGridInterpolator(
        ...,
        fill_value=fill_value,
    )
```